src/cryptoadvance/specter/static/styles.css

```diff
@@ -1282,31 +1291,131 @@ input:checked + .slider:before {
     transform: translateX(26px);
 }
 
-/************** Styles for recipient-box ********************************/
+
+
+
+
+/*****************************/
+/* MESSAGE BOX RELATED CLASS */
+/* (START) */
+/*****************************/
```

Comment:
I'm wondering what difference it makes from a css-point of view whether the notification-system is used or not. Yes, you might be able to use an additional png or so but other than that ... the css here should not be notification-specific?!

Reply:
I have forgotten to move this to the service css file. Will do that.

Reply:
> I have forgotten to move this to the service css file.  

I noticed now that the service css file is of course not available globally....